docs/Concepts/Data-Storage-Formats.md

```diff
@@ -54,12 +54,24 @@ The default limit Bonsai looks back is 512. To change the parameter, use the
     Using `--bonsai-maximum-back-layers-to-load` doesn't affect the size of the database being stored, only how far back to load.
     This means there is no "safe minimum" value to use with this option.
 
-### Fast syncing nodes
+### Syncing nodes
 
-With forest mode, your node can use [fast synchronization](Node-Types.md#run-a-full-node) as normal.
-Your node can fast sync from other nodes and other nodes can fast sync to your node.
+!!! important
 
-You can fast sync from other nodes using Bonsai, but other nodes cannot fast sync
-to a Bonsai node. Bonsai stores data at a point-in-time. Fast synchronization messages request node data by-hash at a
-point-in-time (pivot block), thousands of blocks back during the sync. This limitation will be addressed
-once the [snap-sync protocol](https://github.com/ethereum/devp2p/blob/master/caps/snap.md) is implemented.
+    We recommend using Bonsai with [snap sync](Node-Types.md#snap-synchronization) for the fastest sync and lowest
+    storage requirements.
+
+With forest or Bonsai mode, you can use [fast](Node-Types.md#fast-synchronization) or
+[snap](Node-Types.md#snap-synchronization) sync to run a full node.
+
+With forest mode, your node can fast or snap sync to other nodes and other nodes can fast or snap sync to your node.
```

Comment:
Besu cannot be snapsync as a server in bonsai or forest
Besu can be fastsync as a server as long as we sync our node with fastsync or snapsync but only with forest

next step will be to implement snapsync as a server